### ISSUE TITLE
Snapper can't open .snapshot dirs anymore

### ISSUE THREAD

**Issue by IsaakGroepT** [openSUSE/snapper#454]
I rollbacked and I undochanged subvolume `/` to try to fix a kwallet data loss issue, only to realise later on that that wallet should be stored in `/home`, which is a separate subvolume.

These actions caused minimal data change in the `/` path and no change in the `/var` and `/home` paths, them being different subvolumes.

But now when I try to `sudo snapper -c home list `or `sudo snapper -c var list`, I get these errors:
`IO-fout (open failed path:/home/.snapshots errno:2 (No such file or directory)).`

`IO-fout (open failed path:/var/.snapshots errno:2 (No such file or directory)).`

Nor other rollbacks, undochanges, reboots or whatever solve this issue. The .snapshots dirs with their contents still exist though

**Comment by IsaakGroepT** [comment]
`ls -al` of `/var/.snapshots`

```
totaal 0
drwxr-x--- 1 root root  76 22 nov 21:04 .
drwxr-xr-x 1 root root 136 19 nov 22:40 ..
drwxr-xr-x 1 root root  32 11 sep 12:08 59
drwxr-xr-x 1 root root  32 14 sep 20:39 60
drwxr-xr-x 1 root root  32 14 sep 21:05 61
drwxr-xr-x 1 root root  32 15 sep 08:54 62
drwxr-xr-x 1 root root  32 15 sep 14:03 63
drwxr-xr-x 1 root root  32 16 sep 09:30 64
drwxr-xr-x 1 root root  32 16 sep 11:33 65
drwxr-xr-x 1 root root  32 16 sep 11:35 66
drwxr-xr-x 1 root root  32 17 sep 22:00 67
drwxr-xr-x 1 root root  32 27 sep 21:09 68
drwxr-xr-x 1 root root  32 30 sep 15:40 69
drwxr-xr-x 1 root root  32  7 okt 09:14 70
drwxr-xr-x 1 root root  32  7 okt 09:36 71
drwxr-xr-x 1 root root  32 20 okt 14:26 72
drwxr-xr-x 1 root root  32 30 okt 10:04 73
drwxr-xr-x 1 root root  32 11 nov 20:28 74
drwxr-xr-x 1 root root  32 17 nov 11:17 75
drwxr-xr-x 1 root root  32 19 nov 23:22 76
drwxr-xr-x 1 root root  32 22 nov 21:04 77

```

**Comment by aschnell** [comment]
What version of snapper? Please try version 0.8.1.

**Comment by IsaakGroepT** [comment]
version 0.8.0

**Comment by aschnell** [comment]
Did you try version 0.8.1?

**Comment by IsaakGroepT** [comment]
Not yet, Arch still has version 0.8.0.

**Comment by IsaakGroepT** [comment]
Hi again,

Sorry for the long delay, but the maintainer of the Arch package took a long time to update it. Today he updated the package and I was able to update snapper to 0.8.1.

I tested it and now I am able to access the .snapshots dirs again. I can `sudo snapper -c configfilename list` again, which I couldn't before. Although I don't know which commit caused and solved this issue because it's not clear to see from your way of developing. The last release on github is 0.5.x something so I narrow out the issue. It would be nice to know the details though.
